runtime/interpreter/value.go

```diff
@@ -12186,6 +12385,23 @@ func (v *CompositeValue) Transfer(
 		v.checkInvalidatedResourceUse(getLocationRange)
 	}
 
+	if interpreter.tracingEnabled && v.dictionary != nil {
```

Comment:
I think its safe to assume that `v.dictionary != nil` is already checked in the `checkInvalidatedResourceUse` check.

Reply:
Ah, yes, that was temporary debugging code 👍 
```suggestion
	if interpreter.tracingEnabled {
```